core/templates/pages/exploration-editor-page/statistics-tab/statistics-tab.directive.ts

```diff
@@ -140,7 +140,7 @@ angular.module('oppia').directive('statisticsTab', [
             AlertsService.clearWarnings();
 
             StateInteractionStatsService.computeStats(
-              ExplorationStatesService.getState(stateName)
+              ExplorationStatesService.getState(stateName, false)
```

Comment:
Why false? (I guess I'm thinking, if we have a cache, why wouldn't we use it?)

Perhaps adding a comment might be useful.

Reply:
Changed my mind, I'll just use cached results by-default.